src/typing/typeloadModule.ml

```diff
@@ -196,14 +196,20 @@ let module_pass_1 ctx m tdecls loadp =
 	let pt = ref None in
 	let rec make_decl acc decl =
 		let p = snd decl in
+		let check_type_name name =
```

Comment:
`check_local_variable_name` is located in `Typecore`. This one should probably be moved there too so that both are in the same place.